docs/src/atomicserver/installation.md

```diff
@@ -145,6 +145,52 @@ systemctl restart atomic
 journalctl -u atomic.service --since "1 hour ago" -f
 ```
 
+# Install Atomic Data Server with docker-compose and cloudflare tunnel
+
+To install atomic server with docker-compose and cloudflared tunnel, create docker-compose.yml
+
+```yaml
+version: "3.4"
+
+services:
+  atomic-server:
+    image: joepmeneer/atomic-server
+    container_name: atomic-server
+    restart: unless-stopped
+    environment:
+      ATOMIC_DOMAIN: ${ATOMIC_DOMAIN}
+      ATOMIC_SERVER_URL: ${ATOMIC_SERVER_URL}
+    ports:
+      - 8080:80
+    volumes:
+      - data:/atomic-storage
+  cloudflared:
+    image: cloudflare/cloudflared:latest
+    environment:
+      TUNNEL_URL: ${TUNNEL_URL}
+      TUNNEL_TOKEN: ${TUNNEL_TOKEN}
+    command: "tunnel run"
+    volumes:
+      - ./cloudflared:/etc/cloudflared
+    links:
+      - atomic-server
+    depends_on:
+      - atomic-server
+volumes:
+  data:
+```
+
+and .env file with:
+
+```
+TUNNEL_URL=http://atomic-server:8080
+TUNNEL_TOKEN=op://at.terraphim.dev/token
+ATOMIC_SERVER_URL=op://Shared/at.terraphim.dev/server_url
```

Comment:
We can move configs into sample folder - I don't mind including .env since it's one password URL to fetch data. I think it's easier to have fully functioning docker compose + env, but it depends on secrets, hence 1password thing.